Functions.Templates/Documentation/sqlTrigger.md

```diff
@@ -0,0 +1,36 @@
+#### Settings for an Azure SQL trigger binding
```

Comment:
Is there a reason you made this H4? We should generally start with H1 and go out from there